sample-components/.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+// ESLint config for MagicScript Apps
+module.exports = {
+    env: {
+        "es6": true
+    },
+    globals: {
+        // These globals are provided by the vm itself.
+        "print": true,
+        "globalThis": true,
+        // The following globals are provided by `magic-script-polyfills`
+        "setTimeout": true,
+        "clearTimeout": true,
+        "setInterval": true,
+        "clearInterval": true,
+        "setImmediate": true,
+        "clearImmediate": true,
+        "fetch": true,
+        "Headers": true,
+        "Request": true,
+        "Response": true,
+
```

Comment:
Maybe add `process` here since we are declaring it as a global in one of the source files?

Reply:
I'm not declaring process as a global so this isn't needed